modules/ROOT/pages/dw-core-functions-mapobject.adoc

```diff
@@ -97,3 +97,24 @@ output application/xml
 
 Helper function that allows `mapObject` to work with null values.
 
+=== Example
+
+Using the previous example, you can test that if the input of the `mapObject` is `null`, the output result is `null` as well.
```

Comment:
Note: In XML null values by default are written as empty tags, that can be changed using the writer property `writeNilOnNull=true`

Reply:
Should be this defined under >> **output application/xml writeNilOnNull=true** ? Let me know and I can add this before merging.

The source example should look like this:
%dw 2.0
output application/xml writeNilOnNull=true
---
{
    prices: payload.prices mapObject (value, key) -> {
        (key): (value + 5) as Number {format: "##.00"}
    }
}

Reply:
Should be this defined under `output application/xml writeNilOnNull=true`? Let me know and I can add this before merging.
The source example should look like this:
%dw 2.0
output application/xml writeNilOnNull=true
---
{
    prices: payload.prices mapObject (value, key) -> {
        (key): (value + 5) as Number {format: "##.00"}
    }
}`

Reply:
@leansh Should the Source example be defined as below then?
%dw 2.0
output application/xml **writeNilOnNull=true**
---
{
    prices: payload.prices mapObject (value, key) -> {
        (key): (value + 5) as Number {format: "##.00"}
    }
}

Reply:
No, it's fine as it is, the note was only for clarification of the output (which is a key followed by a null value) and for additional info on how to change this behavior